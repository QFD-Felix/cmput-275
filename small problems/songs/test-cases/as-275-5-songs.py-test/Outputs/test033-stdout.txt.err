0 29 21 3 9 15 40 1 1 2 14 7 14 22 3 18 6 12 13 0 7 0 28 11 19 15 12 1 31 7 7 53 0 18 8 4 20 4 24 27 8 14 2 6 3 0 39 17 6 3 3 21 16 37 7 3 1 3 23 4 13 6 8 11 1 4 39 8 3 34 30 7 8 3 9 2 11 12 1 7 15 1 10 8 4 11 13 31 5 50 4 6 6 11 15 33 19 9 32 44 2 0 5 37 7 8 4 1 6 68 2 16 4 8 6 2 6 6 15 0 11 5 3 1 29 7 37 9 1 5 7 14 37 26 7 0 8 1 10 3 18 24 5 12 5 1 5 50 12 1 6 27 0 20 0 6 1 9 23 3 4 23 18 4 29 10 2 16 4 34 0 0 10 3 0 31 23 2 29 8 11 4 44 3 6 27 2 5 1 7 4 71 13 8 20 0 13 27 0 1 2 10 19 5 0 11 17 3 11 13 34 83 25 0 1 15 1 3 0 17 52 12 5 5 1 10 5 23 12 0 8 1 13 6 0 7 17 29 10 4 9 0 5 3 8 11 8 8 9 35 1 8 16 5 9 10 55 16 2 12 9 0 5 1 27 15 15 38 8 1 4 8 13 5 9 0 24 0 18 25 58 1 5 3 6 8 3 5 0 14 8 4 9 4 26 7 3 22 37 14 8 6 4 21 5 13 53 9 4 1 4 4 15 45 37 6 1 14 11 4 16 1 8 5 16 0 0 5 39 23 3 25 5 9 0 6 17 12 1 8 4 2 3 7 2 26 9 14 0 15 33 23 35 1 16 25 1 3 4 7 13 2 24 0 12 0 3 27 0 6 9 1 9 5 17 21 0 0 4 6 13 0 2 5 13 5 12 2 11 0 0 0 0 0 1 0 0 0 0 1 0 0
