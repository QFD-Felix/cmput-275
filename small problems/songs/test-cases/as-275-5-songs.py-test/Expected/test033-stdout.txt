0 29 21 3 9 15 40 1 1 2 14 7 14 22 3 18 6 12 13 0 7 0 28 11 19 15 12 1 31 7 7 53 0 18 8 4 20 4 24 27 8 14 2 6 3 0 39 17 6 3 3 21 16 37 7 3 1 3 23 4 13 6 8 11 1 4 39 8 3 34 30 7 8 3 9 2 11 12 1 7 15 1 10 8 4 11 13 31 5 50 4 6 6 11 15 33 19 9 32 44 2 0 5 37 7 8 4 1 6 68 2 16 4 8 6 2 6 6 15 0 11 5 3 1 29 7 37 9 1 5 7 14 37 26 7 0 8 1 10 3 18 24 5 12 5 1 5 50 12 1 6 27 0 20 0 6 1 9 23 3 4 23 18 4 29 10 2 16 4 34 0 0 10 3 0 31 23 2 29 8 11 4 44 3 6 27 2 5 1 7 4 72 13 8 20 0 13 27 0 1 2 10 19 5 0 11 17 3 11 13 35 58 25 0 1 15 1 3 0 17 54 12 5 5 1 10 5 24 13 0 9 1 6 7 0 8 18 30 11 7 4 9 0 5 3 8 11 8 8 9 37 1 8 17 5 9 10 59 18 2 13 10 0 5 1 30 17 17 42 10 28 1 5 2 15 6 6 9 0 26 0 20 27 5 1 6 4 7 9 54 3 5 0 15 9 5 10 5 27 0 7 3 22 37 14 8 6 4 21 5 13 53 9 4 1 4 4 15 45 37 6 1 14 11 4 16 1 8 5 16 0 0 5 40 23 3 25 5 9 0 6 17 12 1 8 4 2 3 7 2 27 9 14 0 15 35 24 37 1 17 27 1 3 4 7 14 2 26 14 14 0 4 0 28 0 6 10 1 10 6 18 22 0 0 0 4 6 13 0 2 5 13 5 12 2 11 0 0 0 0 0 1 0 0 0 0 1 0 0
